src/chi-vue/src/components/data-table/DataTableTooltip.tsx

```diff
@@ -22,41 +23,28 @@ export default class DataTableTooltip extends Vue {
 
   _isTruncated() {
     const wrapper = this.$refs.wrapper as HTMLElement;
-    const wrapperClone = wrapper.cloneNode(true) as HTMLElement;
-
-    wrapperClone.style.display = 'inline';
-    wrapperClone.style.width = 'auto';
-    wrapperClone.style.visibility = 'hidden';
-    wrapperClone.style.overflow = 'visible';
-
-    wrapper.parentNode?.appendChild(wrapperClone);
-
-    const isTruncated = wrapperClone.offsetWidth > wrapper.offsetWidth;
-
-    wrapperClone.parentNode?.removeChild(wrapperClone);
+    const isTruncated =
+      Math.abs(wrapper.scrollHeight - wrapper.clientHeight) > DATA_TABLE_CELL_THRESHOLD ||
+      wrapper.scrollWidth > wrapper.clientWidth;
 
     return isTruncated;
   }
 
   render() {
+    const content = (
+      <div class={this.textWrap ? '-cell-wrap' : UTILITY_CLASSES.TYPOGRAPHY.TEXT_TRUNCATE} ref="wrapper">
+        {this.msg}
+      </div>
+    );
+
     return (
       <div
         onMouseenter={() => this.onShow()}
         onFocus={() => this.onShow()}
         onMouseleave={() => this.onHide()}
         onBlur={() => this.onHide()}
         style={`max-width: fit-content; width: ${this.$props.header ? 'calc(100% - 20px)' : '100%'};`}>
-        {!this.tooltip ? (
-          <div class={UTILITY_CLASSES.TYPOGRAPHY.TEXT_TRUNCATE} ref="wrapper">
-            {this.msg}
-          </div>
-        ) : (
-          <Tooltip message={this.msg}>
-            <div class={UTILITY_CLASSES.TYPOGRAPHY.TEXT_TRUNCATE} ref="wrapper">
-              {this.msg}
-            </div>
-          </Tooltip>
-        )}
+        {!this.tooltip ? content : <Tooltip message={this.msg}>{content}</Tooltip>}
```

Comment:
Very minor detail, but I'd suggest to always check on positive conditions, since they are easier to understand
```suggestion
        {this.tooltip ? <Tooltip message={this.msg}>{content}</Tooltip> : content}
```